tap_gitlab/__init__.py

```diff
@@ -248,11 +248,15 @@ def get_start(entity):
         STATE[entity] = CONFIG['start_date']
     return STATE[entity]
 
-
+@backoff.on_predicate(backoff.runtime,
+                      predicate=lambda r: r.status_code == 429,
+                      max_tries = 5, 
```

Comment:
let's keep the styling as consistent as possible

```suggestion
                      max_tries=5, 
```